src/main/java/fishstock/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package fishstock;
+
+import java.time.LocalDateTime;
+
+/**
+ * Encapsulates a Deadline Task.
+ * This Task has a description and deadline date.
+ */
+class Deadline extends Task {
+    protected static final String COMMAND = "deadline";
+    private static final String byKeyword = " /by ";
```

Comment:
Consider renaming `byKeyword` to `BY_KEYWORD` as it is a constant!

Reply:
Thanks for your feedbacks!